src/components/Camera/hooks/useCreateEvidence.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import { CameraRoll } from "@react-native-camera-roll/camera-roll";
+import {
+  useCallback
+} from "react";
+import Observation from "realmModels/Observation";
+import ObservationPhoto from "realmModels/ObservationPhoto";
+import useStore from "stores/useStore";
+
+import { log } from "../../../../react-native-logs.config";
```

Comment:
FWIW, I added a shortcut for this: `import { log } from "sharedHelpers/logger";`